### ISSUE TITLE
BCM94360NG 

### ISSUE THREAD

**Issue by dereek5755** [frostyviking/HP-EliteDesk-800-G2-OC#1]
Hi Sir, 
I'm using your EFI at OS Big Sur as well. with the same WIFI Card " BCM94360NG "

but I got keep WIFI keeps Disconnected and connects back issue. and if the WIFI connected. the connection speed is also extremely slow. 

is that any manual setting you did previously?

same location. all other devices are ok with the same WIFI connection.

**Comment by frostyviking** [comment]
Hello!
I ran into that issue aswell, disconnects and super slow at best.
What I found was in : bios -> advanced -> power management options.
The setting for : **PCI Express Power Management** needs to be **Disabled**.

Worst case, flash your bios to latest version and/or reset bios to defaults.
Then take the bios steps outlined in my readme.md.

HP Machines are pretty good hacks but at the same time -HP do make some funky stuff in their bioses.

I also have a HP Z2 SFF/i9 9000k, running the BCM94360NG card.
But on this machine I can leave the power management setting enabled with no disconnect or slowdowns.
